flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/StaticResultProvider.java

```diff
@@ -51,6 +55,20 @@ public class StaticResultProvider implements ResultProvider {
                         results[i] = "TRUE";
                     } else if (Boolean.FALSE.equals(value)) {
                         results[i] = "FALSE";
+                    } else if (value instanceof GenericMapData) {
```

Comment:
We can change the RowFormat in   `SqlGatewayRestEndpointMaterializedTableITCase#testPeriodicRefreshMaterializedTableViaRestAPI`  to verify this change ?

Reply:
I've verify it manually.